### PR TITLE
Update legrand.js add support LEG600082 and LEG600090

### PR DESCRIPTION
**Pull request by Yann-F**
Adding support for
LEG600082 : double light switch wired with neutral
LEG600090 : single light switch wired with neutral

### CODE REVIEW

File: devices/legrand.js [hunk 1]
```diff
@@ -320,4 +320,66 @@ module.exports = [
             await reporting.activePower(endpoint);
         },
     },
+    {
+        zigbeeModel: ['NLIS - Double light switch'],
+        model: 'LEG600082',
+        vendor: 'Legrand',
+        description: 'Interrupteur double connecté avec Neutre LEG600082',
```
Comment (Koenkk) [inline devices/legrand.js:327]: Could you translate this to English?

**Comment by Koenkk** [comment]
Please fix the linting errors, you can run it locally by executing `npm run lint`.

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/1204

File: devices/legrand.js (hunk 1 above)
Comment (Yann-F) [reply devices/legrand.js]: Yes of course.
Sorry for the mistake.

File: devices/legrand.js [hunk 2]
```diff
@@ -320,4 +320,62 @@ module.exports = [
...
+        configure: async (device, coordinatorEndpoint, logger) => {
+            await extend.light_onoff_brightness().configure(device, coordinatorEndpoint, logger);
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genIdentify', 'genOnOff', 'genLevelCtrl', 'genBinaryInput',
+                'manuSpecificLegrandDevices', 'lightingBallastCfg']);
+            await reporting.onOff(endpoint);
+            await reporting.brightness(endpoint);
+        },
+        onEvent: async (type, data, device) => {
+            await legrand.pairing_security_event(type, data, device);
```
Comment (Koenkk) [inline devices/legrand.js:378]: `pairing_security_event` this method does not exist, I'm not sure why this was added.

Comment (Yann-F) [reply devices/legrand.js:378]: oopss yes was part of a try to support direct greenpower association but doesn't work and I didn't clean correctly my code.
I review my code and fix it.
I'm sorry for that.

**Comment by Koenkk** [comment]
Could you fix the linting issues? You can check it locally by running `npm run lint`

**Comment by vladispavlov** [comment]
@Yann-F I can help you with fixes just allow edits from maintainers. You can read more about this [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

**Pull request closed without merging**